### ISSUE TITLE
Dyslexic font support

### ISSUE THREAD

**Issue by C00kie-** [jupyter/notebook#6962]
### Problem

I wish to use the notebook (and the lab) with dyslexic font. 

### Proposed Solution

Dyslexic font support integrated to notebook.

### Additional context

Even if dyslexic fonts add-on aren't great for security reasons, in this case they can not access the text inside the notebook. 

Thanks!

**Comment by JasonWeill** [comment]
The font in question: https://opendyslexic.org/

**Comment by C00kie-** [comment]
Thanks,
I look forward to it :)

**Comment by JasonWeill** [comment]
Leaving this open; we don't currently embed fonts with Jupyter Notebook, so this would require some development effort.

**Comment by trallard** [comment]
Thanks for taking the time to bring this suggestion @C00kie- 

Some of the Quansight and Jupyter accessibility team folks have been working on an extension containing themes with WCAG conformant colour contrast and using the Atkinson Hyperlegible Typeface by default. 
Such font is also dyslexic and readability focused.
We are planning to have our first stable release in the upcoming weeks https://github.com/Quansight-Labs/jupyterlab-accessible-themes

This might be a good starting point to gather feedback, identify other usability needs, and define which of such settings or features should be in the core of Notebook/JupyterLab and plan for that development and potentially design effort.

**Comment by tonyfast** [comment]
this might be a heavy opinion to bring into to the core of jupyter. there is inconclusive evidence that dyslexic fonts improve the readability of a document. [^personal] while, i don't agree about adding specific font; i do believe in configurability. [jupyterlab-fonts](https://pypi.org/project/jupyterlab-fonts/) addressed this for v3 lab. we should be able to do this better with this new command palette 

studies of speed and accuracy [^Dyslexie][^OpenDyslexic] of dyslexic font usage indicated that features like background color, serifs, text spacing and line spacing can have great impact on readability than font family. these are controls we find in firefox's native reader application (see figure). gains might be made by exposing these text settings more prominently in the notebook configuration.


<figure>

![image](https://github.com/jupyter/notebook/assets/4236275/d4bc3bb2-4961-45a9-b460-d030a8d8a0e0)

<figcaption>firefox reader's native text configuration that modify serifs, whitespace, color and size.</figcaption>
</figure>


i'd imagine that dyslexic users who customize their system fonts to suit their needs would prefer seeing those, overriding font defaults could be a regression for accessibility power users. folks should be able choose a preferred custom font as a fallback though. [webaim] mentions <q cite="https://webaim.org/techniques/fonts/">there is not one typeface that will be optimal for all users with dyslexia.</q> which would encourage configurable fonts to suit needs, maybe even suggestions or help to facilitate choice.

css customization is one of my favorite jupyter features. we can actually override the jupyter font with a dyslexic font using the following cell magic

```html
%%html
<style>
@import url('https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible&display=swap');
:root {
    --jp-ui-font-family: 'Atkinson Hyperlegible', sans-serif;
    --jp-code-font-family: 'Atkinson Hyperlegible', sans-serif;
}
</style> 
```

<figure>

![image](https://github.com/jupyter/notebook/assets/4236275/576942df-a585-475d-962c-6a3ae068df2d)

<figcaption>the jupyterlab interface with atkinson hyperlegible as the font face.</figcaption>
</figure>


[webaim]: https://webaim.org
[^Dyslexie]: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5934461/ "Dyslexie font does not benefit reading in children with or without dyslexia"
[^OpenDyslexic]: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5629233/ "The effect of a specialized dyslexia font, OpenDyslexic, on reading rate and accuracy"

[^personal]: i like atkinson hyperlegible. its a good work-horse.
